src/app/store/machine/selectors.ts

```diff
@@ -386,6 +386,28 @@ const listCount = createSelector(
   (machineState, callId) => getList(machineState, callId)?.count || null
 );
 
+/**
+ * Get the loaded state for a machine list request with a given callId.
+ */
+const listLoaded = createSelector(
+  [
+    machineState,
+    (_state: RootState, callId: string | null | undefined) => callId,
+  ],
+  (machineState, callId) => getList(machineState, callId)?.loaded ?? null
```

Comment:
Do we need the additional state complexity by adding `null`? Couldn't we just default to `false`?

Reply:
In case we want to keep the `null`, would be good to add tests coverage for it.

Reply:
Hmm maybe we don't need this. The `null` case would be if we're trying to select a list that isn't in the store for some reason, but I can't think of a case where we'd need to know that for the loading/loaded states. We can always add this back in if we find we need it.